files/class-api-client.php

```diff
@@ -253,4 +260,20 @@ public function get_unique_filename( $file_path ) {
 
 		return $obj->filename;
 	}
+
+	private function log_request( $path, $method, $rquest_args ): void {
```

Comment:
Minor: typo (and remove the type hint for consistency).

```suggestion
	private function log_request( $path, $method, $request_args ) {
```